reactioncommerce/reaction
# added CDN settings to reaction

Resolves #4315   
Impact: **minor**  
Type: **feature**

## Issue
After creating the production bundle from reaction, if the user wants to serve the bundled js and css files, there is no native way to do it.

## Solution
Meteor provides a function to set the serving URL of the `js` and `css` files, so I am just passing the CDN's URL from setting file to Meteor. 


## Testing
1. Build the production bundle from reaction using `meteor build --directory "build dir"`
1. `cd "build dir"/bundle/programs/server`
1. `npm install`
1. `cd "build dir"/bundle`
1. `PORT=4000 ROOT_URL=http://localhost.com METEOR_SETTINGS="{ \"cdnPrefix\": \"cdnURL\" }" node main.js`
1. Observe from the Network tab in your browser that reaction tries to get the `js` and `css` files from the "cdnURL"
